google-cloud-firestore/src/main/java/com/google/cloud/firestore/FirestoreOptions.java

```diff
@@ -174,6 +181,16 @@ public Builder setCredentialsProvider(@Nonnull CredentialsProvider credentialsPr
       return this;
     }
 
+    /**
+     * Sets the emulator host to use with this Firestore client.
```

Comment:
Can we add a note here explaining the priority this setter has relative to the environment variable?

Something like
```suggestion
     * Sets the emulator host to use with this Firestore client. The value passed to this method will take precedent if the {@code FIRESTORE_EMULATOR_HOST} environment variable is also set.
```